configure.ac

```diff
@@ -55,7 +55,7 @@ case $host in
   ;;
 esac
 dnl Require C++11 compiler (no GNU extensions)
-AX_CXX_COMPILE_STDCXX([11], [noext], [mandatory])
+AX_CXX_COMPILE_STDCXX([11], [noext], [mandatory], [nodefault])
```

Comment:
For this change to work you need to update the respective m4 file for that function
https://github.com/bitcoin/bitcoin/blob/master/build-aux/m4/ax_cxx_compile_stdcxx.m4
Probable the entire `build-aux` directory is worth updating

Reply:
I'd better remove this nodefault. Updating the whole dir would end up in merge conflicts later